.github/workflows/build_rpms.yaml

```diff
@@ -0,0 +1,55 @@
+name: Build RPMs
+
+on:
+  # push:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  build_rpm_build_container:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repository: ManageIQ/manageiq-rpm_build
+    - name: Set up registry credentials
+      if: ${{ github.event_name != 'pull_request' }}
+      run: |
+        echo "REGISTRY_USERNAME=${{ secrets.REGISTRY_USERNAME }}" >> $GITHUB_ENV
+        echo "REGISTRY_PASSWORD=${{ secrets.REGISTRY_PASSWORD }}" >> $GITHUB_ENV
```

Comment:
Do we need a remote registry?  I thought docker build support was built in and I think you have a local registry, which is all we need here.

Reply:
There's just the local copy on the machine it was built on.  If jobs run on different machines they won't have access to the image that was built.  Also, I see little downside to pushing the image to docker.io especially for troubleshooting purposes.